### PR TITLE
test: add bribe tests and test common utilities for liquidity incenti…

### PR DESCRIPTION
**Pull request by swarna1101**
# Add Bribe Tests for Liquidity Incentive Module

## Changes
- Added comprehensive test suite for bribe operations in liquidity incentive module
- Implemented test utilities and mock keepers for streamlined testing
- Added test cases covering bribe creation, updates, and validation

## Approach
- Used table-driven tests for better test coverage and maintainability
- Implemented mock keepers to isolate bribe functionality testing
- Focused on edge cases and validation scenarios

### CODE REVIEW

File: x/liquidityincentive/keeper/msg_server_register_bribe.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func (k msgServer) RegisterBribe(ctx context.Context, msg *types.MsgRegisterBrib
 		return nil, errorsmod.Wrap(err, "failed to get current epoch")
 	}
 
-	if found && msg.EpochId <= currentEpoch.Id {
+	if found && msg.EpochId < currentEpoch.Id {
```
Comment (Senna46) [inline x/liquidityincentive/keeper/msg_server_register_bribe.go:26]: Cannot be created in the current epoch. The process has already been completed.

File: x/liquidityincentive/keeper/testutil/mocks.go [hunk 2]
```diff
@@ -0,0 +1,454 @@
+package testutil
```
Comment (Senna46) [inline x/liquidityincentive/keeper/testutil/mocks.go:1]: use https://github.com/sunriselayer/sunrise/blob/main/x/liquidityincentive/testutil/expected_keepers_mocks.go

File: x/liquidityincentive/keeper/testutil/test_common.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
+package testutil
```
Comment (Senna46) [inline x/liquidityincentive/keeper/testutil/test_common.go:1]: use https://github.com/sunriselayer/sunrise/blob/main/x/liquidityincentive/keeper/keeper_test.go

**Review by Senna46 — CHANGES_REQUESTED** [review]
